boot/zephyr/Kconfig

```diff
@@ -120,6 +120,11 @@ config BOOT_SIGNATURE_KEY_FILE
 	  with the public key information will be written in a format expected by
 	  MCUboot.
 
+config CLEANUP_ARM_CORE
```

Comment:
As this is defined in mcuboot, I assume it will only be used in mcuboot source code, should we prefix this with "MCUBOOT_" ?

Reply:
prefix added